packages/code-highlighting/fixtures/_read-file.js

```diff
@@ -1,8 +1,10 @@
-const fs = require('fs')
-const path = require('path')
+import fs from 'fs'
+import path from 'path'
+import { fileURLToPath } from 'url'
 
-function readFile(file) {
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
```

Comment:
`__dirname` is not available in ESM, so gotta do this.

Reply:
wow really? this is nuts

Reply:
Yeah, some of the little paper cuts like this are head scratchers. I'm sure there's a reasonable academic or theoretical reason for it but....why? 😅